opencensus/stats/internal/view_data_impl.h

```diff
@@ -106,6 +107,9 @@ class ViewDataImpl {
   absl::Time start_time() const { return start_time_; }
   absl::Time end_time() const { return end_time_; }
```

Comment:
Can we remove end_time from the view_data_impl? End time is always the moment when we read the data, because we read all the updates until that moment (I know it is an approximation, because we do have some async logic, but it is good enough).

Reply:
We would have to also remove end_time from the public API in `opencensus/stats/view_data.h`

```c++
absl::Time ViewData::end_time() const { return impl_->end_time(); }
```

Reply:
That will be the value from the getdata?

Reply:
Ok, so do you mean:
* Keep the public API `ViewData::end_time` and
* Replace internal `ViewDataImpl::end_time` with a new member `ViewData::end_time_`